index.html

```diff
@@ -190,10 +229,10 @@ <h2>Submitting for Review</h2>
 			<a name="Deleting"><div class="section">
 				<h2>Deleting a Learning Object</h2>
 				<hr>
-				<p>Go to your learning objects dashboard. From here click on the meatball menu, same as you would if you wanted to edit or submit an object for review. There is an option to delete that individual learning object. The second way is to delete multiple at once. To do this check each learning object you want to delete then click on the delete button at the top of the page.</p>
+				<p>Go to your learning objects dashboard. From here click on the meatball menu, same as you would if you wanted to edit or submit an object for review. There are two options to delete a learning object that you have created. There is an option to delete that individual learning object. The second way is to delete multiple at once. To do this check each learning object you want to delete then click on the delete button at the top of the list.</p>
```

Comment:
I would add you can't delete a learning object that is not unreleased.  You can delete unreleased drafts and revision drafts but not if the object is in-review or released.